python/mlc_chat/compiler/loader/huggingface_loader.py

```diff
@@ -14,16 +14,43 @@
 
 from .mapping import ExternMapping, QuantizeMapping
 from .stats import Stats
-from .utils import (
-    ParamQuantizer,
-    check_parameter_usage,
-    load_safetensor_shard,
-    load_torch_shard,
-)
+from .utils import check_parameter_usage, load_safetensor_shard, load_torch_shard
 
 logger = logging.getLogger(__name__)
 
 
+class ParamQuantizer:
```

Comment:
Given this `ParamQuantizer` applies to all the loaders given `quantize_map`, would it be better if we keep it in a different location?

Reply:
I will refactor this into a method instead of a class, and put it back to utils.py